schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -102,6 +102,11 @@
           "type": "boolean",
           "default": false
         },
+        "suppressInstallNotes": {
```

Comment:
Not sure it applies here, but I remember seeing recommendations of using only positive sentences to avoid confusion with double negatives. Like "do show notes" instead of "don't suppress notes".